pkgs/tools/misc/mongodb-tools/goconvey.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, goPackages, fetchFromGitHub, callPackage }:
```

Comment:
Also openssl and the other go package. They must go in goPackages, and try to be shared if psosible instead of having multiple versions of it.